### PR TITLE
Vessel Viewer search changes

### PR DESCRIPTION
**Pull request by satellitestudiodesign**
- [x] Remove simple search
- [x] Add a "confirm" button
- [x] Add transmissionsTimeline to results

Screenshot 2021-09-10 at 12 39 48

### CODE REVIEW

**Review by fertandil87 — APPROVED** [review]

File: applications/vessel-history/src/features/search/AdvancedSearch.tsx [hunk 1]
```diff
@@ -78,30 +75,34 @@ const AdvancedSearch: React.FC = () => {
...
         <InputText
+          inputSize="small"
           onChange={onMainQueryChange}
           value={query}
           label={t('search.shipname', 'Name')}
           autoFocus
           className={styles.half}
         />
         <InputText
+          inputSize="small"
```
Comment (rdgfuentes) [inline applications/vessel-history/src/features/search/AdvancedSearch.tsx:89]: The `width` of this field is too small, mmsi's have a lenght of 8 and it's currently not being displayed completely. I entered `12345678` for the mmsi but I see only up to the 7.
image

File: applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx [hunk 2]
```diff
@@ -94,6 +94,13 @@ const VesselListItem: React.FC<ListItemProps> = (props): React.ReactElement => {
             ) : (
               DEFAULT_EMPTY_VALUE
             )}
+            {vessel.firstTransmissionDate && vessel.lastTransmissionDate && (
+              <TransmissionsTimeline
+                firstTransmissionDate={vessel.firstTransmissionDate}
+                lastTransmissionDate={vessel.lastTransmissionDate}
+                firstYearOfData={FIRST_YEAR_OF_DATA}
+              />
+            )}
```
Comment (rdgfuentes) [inline applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx:103]: This looks awesome! 😎  thanks for adding it!

File: applications/vessel-history/src/features/search/search.slice.ts [hunk 3]
```diff
@@ -24,8 +24,6 @@ export type SearchSlice = {
   queries: CachedQuerySearch
 }
 
-export type SearchType = 'basic' | 'advanced'
```
Comment (rdgfuentes) [inline applications/vessel-history/src/features/search/search.slice.ts:27]: 👍  should we also remove it from the workspace params https://github.com/GlobalFishingWatch/frontend/blob/3807ba41c972052650d0ad524c96407bc4da85ad/applications/vessel-history/src/types/index.ts#L6 ?

**Review by rdgfuentes — COMMENTED** [review]
Something is not working as expected, the first time you search for a vessel by name (monica) you have to click `SEARCH` button to see the results, then you search for another vessel (inter nord) and click `SEARCH` button to see the results, that's how it should work by design 👍 . However if you search again for the first vessel (monica), then the results appear right after you type the last letter of the vessel name without clicking the `SEARCH` button.

https://user-images.githubusercontent.com/2205831/133098931-5943355e-6da8-471d-9a16-d8846f655044.mov

**Review by rdgfuentes — APPROVED** [review]

**Pull request merged**